opm/autodiff/StandardWell_impl.hpp

```diff
@@ -632,7 +632,9 @@ namespace Opm
 
         // add vol * dF/dt + Q to the well equations;
         for (int componentIdx = 0; componentIdx < numWellConservationEq; ++componentIdx) {
-            EvalWell resWell_loc = (wellSurfaceVolumeFraction(componentIdx) - F0_[componentIdx]) * volume / dt;
+            // TODO: following the development in MSW, we need to convert the volume of the wellbore to be surface volume
+            // since all the rates are under surface condition
+            EvalWell resWell_loc = (F0_[componentIdx] - wellSurfaceVolumeFraction(componentIdx)) * volume / dt;
             resWell_loc += getQs(componentIdx) * well_efficiency_factor_;
```

Comment:
Thanks for the fix.

Reply:
With some discussion with @atgeirr and @hnil , we think it is good to make the code in the form of (A-A0)/dt for ease of reading, which means we need to change the sign in other places. 

The current PR is just the way to do it with the least code change. 

I will revise this PR following the discussion.